DEVELOPERS.md

```diff
@@ -214,6 +217,24 @@ Vite builds the assets and outputs them to the `assets/dist` folder.
 
 [Django Staticfiles app](https://docs.djangoproject.com/en/3.2/ref/contrib/staticfiles/) then collects the files and places them in the `staticfiles/assets` folder, with the manifest file located at `staticfiles/manifest.json`.
 
+### Data Setup
+Sometimes it's useful to have a fresh local installation or you may not have authorization to download a production backup.
+In that situation you can follow the steps below to set up your local copy of the site:
+
+1. Create [a GitHub OAuth application](https://github.com/settings/applications/new), the fields don't matter too much for local dev, except the callback URL which needs to be `http://localhost:8000/complete/github/`.  Then set the `SOCIAL_AUTH_GITHUB_KEY` and `SOCIAL_AUTH_GITHUB_KEY` environment variables with values from that application.
```

Comment:
nit:

> , the fields

is a run-on sentence.

suggestion: You could break this up a bit more to make the individual steps stand out:

```suggestion
1. Create [a GitHub OAuth application](https://github.com/settings/applications/new).
   * The callback URL must be `http://localhost:8000/complete/github/`.
   * The other fields don't matter too much for local development.
1. Set the `SOCIAL_AUTH_GITHUB_KEY` and `SOCIAL_AUTH_GITHUB_KEY` environment variables with values from that OAuth application.
```

(Best to check the list formatting works here, because nested lists are always a pain to get the indentation right when I can't directly preview how this looks)